src/middlewared/middlewared/plugins/catalogs_linux/items.py

```diff
@@ -23,6 +23,9 @@ class Config:
         Dict(
             'options',
             Bool('cache', default=True),
+            Bool('retrieve_all_trains', default=True),
+            Bool('skip_retrieving_versions', default=False),
```

Comment:
Can we make it `Bool('retrieve_versions', default=True),` to avoid double negative?